From 78431a361bed0b6e9f1f3ebc0de8897b069b3848 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Wed, 15 Jan 2020 17:54:26 +0000
Subject: [PATCH] autotools: unbreak --enable-wayland with BSD sed

BSD sed doesn't support inlining argument for `i` command, so embed
newlines as proposed in http://austingroupbugs.net/view.php?id=249

  GEN      wayland-drm-client-protocol.c
sed: 1: "1i#include "sysdeps.h"
": command i expects \ followed by text
---
 va/wayland/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/va/wayland/Makefile.am b/va/wayland/Makefile.am
index 705efa3d..f48afb4d 100644
--- a/va/wayland/Makefile.am
+++ b/va/wayland/Makefile.am
@@ -70,7 +70,7 @@ va_wayland_drm.c: $(protocol_source_h)
 %-client-protocol-export.c : %.xml
 	$(AM_V_GEN)$(WAYLAND_SCANNER) code < $< > $@
 %-client-protocol.c: %-client-protocol-export.c
-	$(AM_V_GEN)$(SED) -e '1i#include "sysdeps.h"' \
+	$(AM_V_GEN)$(SED) -e $$'1i\\\n#include "sysdeps.h"\n' \
 	                   -e 's@WL_EXPORT@DLL_HIDDEN@g' < $< > $@
 
 EXTRA_DIST = \
